### PR TITLE
Communicate errors

### PR DESCRIPTION
**Pull request by IongIer**

### CODE REVIEW

File: apis/src/pages/register.rs [hunk 1]
```diff
@@ -1,68 +1,203 @@
...
                         autocomplete="email"
                         placeholder="Email"
                     />
+                    <p class="h-5">
+                        <Show when=is_invalid_email>
+                            <small class="text-ladybug-red ">"Invalid email"</small>
+                        </Show>
+                    </p>
+                    <small>
+                        We will use it for password reset(when we eventually implement it)
```
Comment (klautcomputing) [inline apis/src/pages/register.rs:144]: Notifications and email verification as well?

File: apis/src/pages/register.rs [hunk 2]
```diff
@@ -1,68 +1,203 @@
...
+    let pw_confirm = RwSignal::new(String::new());
+    let has_invalid_char = RwSignal::new(false);
+    let is_invalid_email = RwSignal::new(false);
+    let throttled_input = move |signal_to_update| {
+        use_debounce_fn_with_arg(update_from_input(signal_to_update), 350.0)
+    };
 
+    let pw_invalid = move || {
+        let pw = pw();
+        !(7 < pw.len() && pw == pw_confirm())
```
Comment (klautcomputing) [inline apis/src/pages/register.rs:43]: We should check here that it's not greater than 128 chars :)

**Review by klautcomputing — CHANGES_REQUESTED** [review]
Nothing major just some small suggestions

File: apis/src/pages/register.rs (hunk 2 above)
Comment (IongIer) [reply apis/src/pages/register.rs:43]: the form prevents you from going over, isn't that enough?

**Review by klautcomputing — APPROVED** [review]

**Pull request merged**